plugins/text-freetype2/text-freetype2.c

```diff
@@ -1,4 +1,4 @@
-﻿/******************************************************************************
+/******************************************************************************
```

Comment:
I assume your editor added a BOM.  We don't like BOMs around these parts.

Reply:
My editor actually removed a BOM in this instance.
`
-<EF><BB><BF>/******************************************************************************
+/******************************************************************************
`

Reply:
Ah, whoops.  Not a biggie then.

Reply:
But what if someone tried to open the file on a PowerPC?!